### ISSUE TITLE
Figure 6: Low rank approximation of low rank

### ISSUE THREAD

**Issue by dpmcsuss** [TangRunze/LLG#2]
This figure seems a little too obvious to include but happy to be vetoed.
Also, while it may be true that "it is possible that the true but unknown population mean has the low-rank property but the entry-wise sample mean graph fails to capture it" this is highly unlikely, since the entry-wise variance for the 454 is low enough to capture the overall structure. (I also tried this on >2000 graphs from all of neurodata and same store).

See https://github.com/TangRunze/LLG/blob/master/plos-latex/neuroimage_revise.tex#L678 and the preceding discussion.

**Comment by dpmcsuss** [comment]
Also, Runze, I think i led you astray. That is showing the nuclear norm (sum of abs of evals) rather than the frobenius norm. (I'll update to have Frobenius norm which will make the data look more low-rank ....) My bad.

**Comment by dpmcsuss** [comment]
Based on the new plots (see commit 3624c9811272f7f8ef7d0967a605f2dc44d6abc7) I think its more reasonable to say that these are low-ish rank (though I still think the histogram of entries for P is informative for understanding why it doesn't help as much as one might hope.)

**Comment by TangRunze** [comment]
I agree the figure is very obvious, I am happy to remove it. Let's if jovo thinks neuroscientists need it or not.
